src/libImaging/Draw.c

```diff
@@ -434,7 +446,7 @@ draw_horizontal_lines(
                 }
             }
 
-            (*hline)(im, xmin, e[i].ymin, xmax, ink);
+            (*hline)(im, xmin, e[i]->ymin, xmax, ink);
```

Comment:
Could you explain your thinking behind this change?

Reply:
@radarhere 
Hi, I'm thinking using `Edge **edge_table` to store "horizontal edges" and sort them.
(Currently `edge_table` only store "non-horizontal edges", and there is space for horizontal ones)

The reason why I want to sort them is that, if the order of continuous horizontal edges is (x-value) descending, 
the edge whose x is bigger will be skipped and never be drawn.

Since it's pointer of pointer, it's needed to use `->` instead of `.`

https://github.com/python-pillow/Pillow/pull/5240/files#diff-09507fd39d242b056864e569bedf77dda352959bad5f0fa7acd077c8d9371dcaR460

Reply:
An example of the case:
https://github.com/python-pillow/Pillow/issues/5235#issuecomment-770226489